dockerfiles/redis/Dockerfile

```diff
@@ -0,0 +1,7 @@
+
+FROM rust:1.36.0 
+COPY Cargo.lock /app/Cargo.lock 

```

Comment:
Why is this file placed in `dockerfiles/redis/Dockerfile`? Shoudn't it be named like the others? 

<img width="1027" alt="Screenshot 2020-06-07 at 5 06 48 PM" src="https://user-images.githubusercontent.com/3893573/83967666-537c8980-a8e1-11ea-860f-f255fc633fa9.png">

In this case, I think it should be `dockerfiles/redis/rust-1.36.Dockerfile`.

Reply:
Yes sir I am updating it